src/liquid/components/ld-card/ld-card.tsx

```diff
@@ -23,7 +23,7 @@ export class LdCard {
   @Prop() shadowInteractive?: 'active' | 'hover' | 'stacked' | 'sticky'
 
   /** The size prop effects the card padding only. */
-  @Prop() size?: 'sm'
+  @Prop() size?: 'sm' | 'zero-padding'
```

Comment:
Naming  is tricky. 
First I was confused about using size a property as I'd expected it to basically define the dimensions of the card, not the padding.

As size already was solely for padding I'd at least stick to a naming pattern. Meaning either:
- sm-padding
- zero-padding

or

- sm
- zero

What do you think?

Reply:
I'll change to zero.